packages/graphql/pubspec.yaml

```diff
@@ -20,6 +20,7 @@ dependencies:
   rxdart: ^0.22.0
   websocket: ^0.0.5
   quiver: ">=2.0.0 <3.0.0"
+  path_provider: ^1.0.0
```

Comment:
I thought depending on `path_provider` makes a package web-incompatible? I think we'll need to move the `dart:io`-specific caching stuff to a 3rd packages

Reply:
From my own tests, it doesn't seem to be the case. As long as it is not being used by flutter for web project, it will work. It seems they have improved it a bit, and it follows import paths instead. This allows you to use conditional imports for both `dart:io` and `dart:html` on the same project.